src/tools/perf/cuda/cuda_alloc.c

```diff
@@ -139,29 +143,13 @@ static void ucx_perf_cuda_memcpy(void *dst, ucs_memory_type_t dst_mem_type,
                                  const void *src, ucs_memory_type_t src_mem_type,
                                  size_t count)
 {
-    cudaError_t cerr;
-
-    cerr = cudaMemcpy(dst, src, count, cudaMemcpyDefault);
-    if (cerr != cudaSuccess) {
-        ucs_error("failed to copy memory: %s", cudaGetErrorString(cerr));
-    }
-
-    cerr = cudaDeviceSynchronize();
-    if (cerr != cudaSuccess) {
-        ucs_error("failed to sync device: %s", cudaGetErrorString(cerr));
-    }
+    CUDA_CALL(, cudaMemcpy, dst, src, count, cudaMemcpyDefault);
+    CUDA_CALL(, cudaDeviceSynchronize);
 }
 
-static void* ucx_perf_cuda_memset(void *dst, int value, size_t count)
+static void ucx_perf_cuda_memset(void *dst, int value, size_t count)
 {
-    cudaError_t cerr;
-
-    cerr = cudaMemset(dst, value, count);
-    if (cerr != cudaSuccess) {
-        ucs_error("failed to set memory: %s", cudaGetErrorString(cerr));
-    }
-
-    return dst;
+    CUDA_CALL(, cudaMemset, dst, value, count);
```

Comment:
Maybe keep the signature since it replicates C library `memset`.
And change to
```suggestion
    CUDA_CALL(dst, cudaMemset, dst, value, count);
    return dst;
```

Reply:
fixed